checks.d/linux_proc_extras.py

```diff
@@ -24,47 +25,66 @@
 }
 
 class MoreUnixCheck(AgentCheck):
+    tags = []
+    proc_path_map = {
```

Comment:
It's also static, the tags thing was wrong as it's not static, but this makes sense to include here